packages/main/src/plugin/statusbar/statusbar-item.ts

```diff
@@ -74,6 +75,15 @@ export class StatusBarItemImpl implements StatusBarItem {
     this.update();
   }
 
+  public set badged(badged: boolean) {
+    this._badged = badged;
+    this.update();
+  }
+
+  public get badged(): boolean {
+    return this._badged;
+  }
+
```

Comment:
I would suggest to use another term than badge as badge would be more the overall entry element (also we have a BadgeUI element which is displaying small letters inside so it might be confusing to have "badges" and "badged"

maybe dot ? as we display a dot/circle ?